sphinx/source/docs/user_guide/server.rst

```diff
@@ -278,31 +272,43 @@ The full set of files that Bokeh server knows about is:
        +---templates
             +---index.html
 
-The optional components are
+Here are this tree's optional components:
```

Comment:
I'm not sure everyone will immediately know what "tree" is in this context , also the optional components are really optional parts of the *application*, which just happen to be expressed by files in the directory (or not)